include/overlay005/ov5_021DFB54.h

```diff
@@ -20,7 +20,7 @@ void ov5_021E0E94(PlayerAvatar * playerAvatar);
 void ov5_021E0EEC(PlayerAvatar * playerAvatar);
 SysTask * ov5_021E1000(FieldSystem * fieldSystem);
 void ov5_021E100C(SysTask * param0);
-SysTask * ov5_021E1014(FieldSystem * fieldSystem);
-void ov5_021E1020(SysTask * param0);
+SysTask * Player_SetStateVsSeeker(FieldSystem * fieldSystem);
+void Player_ResetVsSeekerState(SysTask * param0);
```

Comment:
Player_ prefixes aren't appropriate for these two funcs since they take in FieldSystem and SysTask respectively (where PlayerAvatar is expected)

Reply:
So do I use PlayerAvatar_ now or FieldSystem_/SysTask_ ?

Reply:
`FieldSystem_StartVsSeekerTask` and `FieldSystem_FinishVsSeekerTask`?